README.md

```diff
@@ -124,7 +124,7 @@ This command will:
 
 Steps:
 
-1. Download the binary, config and genesis files from [release](https://github.com/binance-chain/bsc/releases/download/v1.0.0-alpha.0/binary.zip), or compile the binary by `make geth`. 
+1. Download the binary, config and genesis files from [release](https://github.com/bnb-chain/bsc/releases/download/v1.1.8/mainnet.zip), or compile the binary by `make geth`. 
```

Comment:
Here is to download the binary, while mainnet.zip is just config.

Reply:
updated